gcloud/pubsub/client.py

```diff
@@ -43,6 +46,28 @@ class Client(JSONClient):
     """
 
     _connection_class = Connection
+    _publisher_api = _subscriber_api = _iam_policy_api = None
+
+    @property
+    def publisher_api(self):
+        """Helper for publisher-related API calls."""
+        if self._publisher_api is None:
+            self._publisher_api = _PublisherAPI(self.connection)
```

Comment:
I'm anticipating the `gRPC` stuff, where these properties will end up creating objects which hold stubs:  I'm assuming that creating stubs is expensive, and thus worth deferring.  I don't really understand the value of the "create in `start`" pattern in Bigtable, actually:  why would application code *want* to call `start`?